lib/minimap-element.js

```diff
@@ -410,7 +377,7 @@ class MinimapElement {
     this.attachedToTextEditor = this.queryParentSelector("atom-text-editor") === this.minimap.getTextEditorElement()
 
     if (this.attachedToTextEditor) {
-      this.minimap.getTextEditorElement().setAttribute("with-minimap", "")
+      this.minimap.getTextEditorElement().setAttribute("with-minimap", this.displayMinimapOnLeft ? "left" : "right")
```

Comment:
Turns out this actually wasn't needed for this fix but I wasn't sure if there was a reason not to have this. It might make other things easier to know if the minimap is on the left or right given the text editor.

Reply:
Yeah, it does not hurt. We are already setting the attribute, so it might be better to include more information.